sonar-db/src/test/java/org/sonar/db/version/v50/FeedFileSourcesTest.java

```diff
@@ -89,6 +89,43 @@ public void migrate_ignores_duplicate_multiple_rows_in_snapshot_source_for_the_s
   }
 
   @Test
+  public void migrate_ignores_duplicate_multiple_rows_in_snapshots_with_islast_true() throws Exception {
+    db.prepareDbUnit(getClass(), "before.xml");
+    /*
+      <snapshots id="6" project_id="3" parent_snapshot_id="4" root_project_id="1" root_snapshot_id="2"
+             status="P" islast="[true]" purge_status="1"
+             period1_mode="days1" period1_param="30" period1_date="2011-09-24"
+             period2_mode="days2" period2_param="31" period2_date="2011-09-25"
+             period3_mode="days3" period3_param="32" period3_date="2011-09-26"
+             period4_mode="days4" period4_param="33" period4_date="2011-09-27"
+             period5_mode="days5" period5_param="34" period5_date="2011-09-28"
+             depth="1" scope="FIL" qualifier="FIL" created_at="2008-12-02" build_date="2011-09-29"
+             version="2.1-SNAPSHOT" path="1.2."/>
+     */
```

Comment:
nope, I copied/pasted the XML to write it's java replacement and forgot to remove it